framework/web/CompositeUrlRule.php

```diff
@@ -68,13 +73,62 @@ public function parseRequest($manager, $request)
      */
     public function createUrl($manager, $route, $params)
     {
-        foreach ($this->rules as $rule) {
-            /* @var $rule \yii\web\UrlRule */
-            if (($url = $rule->createUrl($manager, $route, $params)) !== false) {
+        $this->createStatus = UrlRule::CREATE_STATUS_SUCCESS;
+        $url = $this->iterateRules($this->rules, $manager, $route, $params);
+        if ($url !== false) {
+            return $url;
+        }
+
+        if ($this->createStatus === UrlRule::CREATE_STATUS_SUCCESS) {
+            // create status was not changed - there is no rules configured
+            $this->createStatus = UrlRule::CREATE_STATUS_PARSING_ONLY;
+        }
+        return false;
+    }
+
+    /**
+     * Iterates through specified rules and call [[createUrl()]] for each of them.
```

Comment:
Iterates through specified rules and call → Iterates through specified rules and calls